### DISCUSSION TITLE
Ability To Have An Image On Printout

### DISCUSSION THREAD

**Discussion by enchant97** [mealie-recipes/mealie#1576] [Feature Request]
It would be nice to have an optional image for when printing a recipe. Maybe you could customise the size of the image?

**Comment by michael-genson** [comment]
I think this is a good idea! Are there any other customizations you/anyone else might want on the print page? I'm thinking we can make a small "print options" menu.

**Reply by enchant97** [reply]
I'm migrating from CookBook, which has a small image in the top left corner of the page. For me personally I don't want a very large image as it would waste ink. But maybe the ability to change the image position between left, centre and right might be useful.

**Reply by hay-kot** [reply]
@michael-genson, checkout the print options on the King Arthur page, it has a great set of features I've wanted to implement, but never got around too, might give you some ideas

https://www.kingarthurbaking.com/recipes/ginger-scallion-lobster-baos-recipe (click print)

**Reply by michael-genson** [reply]
Oh wow that's really nice, it's basically got a print preview popup with a settings header. I'll see if I can figure it out... :)

**Comment by michael-genson** [comment]
FYI this has been implemented: https://github.com/mealie-recipes/mealie/pull/2131
